backend/app/services/config_service.py

```
@@ -0,0 +1,168 @@
+        cursor.execute(
+            """
+            INSERT OR REPLACE INTO site_configurations (domain, config_json)
+            VALUES (?, ?)
+        """,
+            (config.domain, config_json),
+        )
+
+        db_channel.commit()
+        db_channel.close()
+
+    def learn_field_mappings(
```

Ruff (0.12.2)

Missing return type annotation for public function `learn_field_mappings`

Add return type annotation: `None`

(ANN201)